public/app/features/alerting/unified/components/silences/SilencedInstancesPreview.tsx

```diff
@@ -43,18 +65,18 @@ export const SilencedInstancesPreview = ({ amSourceName, matchers }: Props) => {
   return (
     <div>
       <h4 className={styles.title}>
-        Silenced alert instances
+        Affected alert rule instances
```

Comment:
Doesn't the text imply that we only display instances of a single rule?
Maybe we should display a slightly different text depending on the presence of the `ruleUid` prop?

Reply:
To me that reads as "instances across one or many alert rules" - or is that not accurate in all scenarios?

Reply:
Then it's okay :)